tests/unit/test_entrypoint.py

```diff
@@ -14,23 +14,21 @@
 #
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see https://www.gnu.org/licenses/.
-"""Test suite for the entrypoint function and helpers."""
+"""Test suite for the cli function and helpers."""
```

Comment:
I think the function name is still "entrypoint". The module is named "cli", though.

Reply:
Thanks, I also renamed the file.